### PR TITLE
Remove cockpit-testing.service and cockpit-testing.socket

### PR DESCRIPTION
**Pull request by stefwalter**
Use the standard cockpit.service and cockpit.socket during testing,
with the exception of disabling TLS because of phantomjs 1.9

### CODE REVIEW

File: test/testvm.py [hunk 1]
```diff
@@ -1219,7 +1219,7 @@ def start_cockpit(self, atomic_wait_for_host="localhost"):
                 self.execute(script=RUN_COCKPIT_CONTAINER)
                 self.wait_for_cockpit_running(atomic_wait_for_host)
         else:
-            self.execute("systemctl start cockpit-testing.socket")
+            self.execute("mkdir -p /etc/systemd/system/cockpit.service.d/ && printf \"[Service]\nExecStartPre=-/bin/sh -c 'echo 0 > /proc/sys/kernel/yama/ptrace_scope'\nExecStart=\nExecStart=/usr/libexec/cockpit-ws --no-tls\n\" > /etc/systemd/system/cockpit.service.d/notls.conf && systemctl daemon-reload && systemctl start cockpit.socket")
```
Comment (dperpeet) [inline test/testvm.py]: Can we make this a script (file) or a multiline string somehow? It's pretty long and will be difficult to read if a small part changes.

Comment (stefwalter) [inline test/testvm.py]: Changed to a multiline string, and added more logic in this area.

**Comment by dperpeet** [comment]
rhel-7 has several failures, e.g.:
```
# ----------------------------------------------------------------------
# testBasic (check_connection.TestConnection)
#
> fatal: 0 error
Job for cockpit.socket failed. See "systemctl status cockpit.socket" and "journalctl -xe" for details.
Wrote TestConnection-testBasic-FAIL.png
Journal database copied to TestConnection-testBasic-10.111.123.244-FAIL.journal
Journal extracted to TestConnection-testBasic-10.111.123.244-FAIL.log
not ok 34 testBasic (check_connection.TestConnection)
Traceback (most recent call last):
  File "./verify/check-connection", line 42, in testBasic
    m.start_cockpit()
  File "/build/cockpit/test/testvm.py", line 1236, in start_cockpit
    """)
  File "/build/cockpit/test/testvm.py", line 299, in execute
    raise subprocess.CalledProcessError(proc.returncode, command, output=output)
CalledProcessError: Command '<script>' returned non-zero exit status 1
```

**Comment by stefwalter** [comment]
Fixed the Debian issue, I hope that gets us further. Will keep an eye on this.

**Pull request closed without merging**